app.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
We don't need to deploy deprecated/frozen docs to Heroku anymore, so this can probably go.